inst/model/nonmem/basic/1.ctl

```diff
@@ -1,7 +1,8 @@
 $PROBLEM PK model 1 cmt base
 
-$INPUT ID TIME MDV EVID DV AMT  SEX WT ETN
+$INPUT ID TIME MDV EVID DV AMT  SEX WT ETN NUM
 $DATA ../../../../extdata/acop.csv IGNORE=@
+IGNORE(ID.EQ.2)
```

Comment:
Hmm, so presumably the motivation for these changes will be clear once the test for the new helpers arrive?

Reply:
heh, yea that's right. This is ignoring one of the subjects in the trial data, essentially so that we can test this functionality: https://github.com/metrumresearchgroup/bbr/blob/bfb6eb2df69ae7bc2e250aee471b748bd0680695/tests/testthat/test-nm-join.R#L16-L37